tests/components/withings/snapshots/test_sensor.ambr

```diff
@@ -2260,14 +2266,14 @@
       'device_class': 'duration',
       'friendly_name': 'henk Light sleep',
       'state_class': <SensorStateClass.MEASUREMENT: 'measurement'>,
-      'unit_of_measurement': <UnitOfTime.SECONDS: 's'>,
+      'unit_of_measurement': <UnitOfTime.HOURS: 'h'>,
     }),
     'context': <ANY>,
     'entity_id': 'sensor.henk_light_sleep',
     'last_changed': <ANY>,
     'last_reported': <ANY>,
     'last_updated': <ANY>,
-    'state': '10440',
+    'state': '2.900',
```

Comment:
Should we update the default precision for these?

Reply:
Changing the display precision is a bit weird on these. On one side, they are show as `1h 20m` on the UI, so the precision does not seem to affect that.

On the other hand, [the logic we have on the sensor state](https://github.com/home-assistant/core/blob/withings_units/homeassistant/components/sensor/__init__.py#L673-L673) makes it so that for the value `0.529722222222222`:
- if no precision is set, we round `state` to `0.530`
- if a precision is set, we return the `state` as `0.529722222222222`

(the rounding depends on the from/to units)


So I am not sure if it makes sense to set a display precision on these.

Reply:
Oh if we display it like that this is fine